src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -324,16 +324,30 @@ private static void makePublic(final CtClass c) {
   }
 
   /**
-   * This forces the creation of an instance for a given class. If the
-   * class provides a public default constructor, it is called. If the class has
-   * a private default constructor, it is made accessible and is then called.
+   * This forces the creation of an instance for a given class. If the class
+   * provides a public default constructor, it is called. If the class has a
+   * private default constructor, it is made accessible and is then called. Else
+   * we use Objenesis to force creation of an instance.
+   *
+   * There are several exceptions that `Constructor.netInstance` can throw. We
+   * catch `IllegalAccessException` and `InstantiationException` and continue
+   * with Objenesis in those cases. `IllegalArgumentException` cannot be thrown
+   * as we only try to use the default constructor which has no parameters. An
+   * `InvocationTargetException` signals an exception in the constructor and is
+   * passed to the calling method. In the case of an
```

Comment:
These last three lines of the documentation (and possible the whole paragraph) should be moved to `Reflector.getInstance(String)`, which is the method that actually deals with the `ExceptionInInitializerError`.

Reply:
done